src/rashdf/plan.py

```diff
@@ -25,6 +25,38 @@ class RasPlanHdfError(Exception):
     pass
 
 
+class XsSteadyOutputVar(Enum):
+    """Summary of steady cross section output variables."""
+
+    ENERGY_GRADE = "Energy Grade"
+    FLOW = "Flow"
+    WATER_SURFACE = f"Water Surface"
+
+
+XS_STEADY_OUTPUT = [
+    XsSteadyOutputVar.WATER_SURFACE,
+    XsSteadyOutputVar.ENERGY_GRADE,
+    XsSteadyOutputVar.FLOW,
+]
+
+
+class XsSteadyAdditionalVar(Enum):
+    """Summary of steady cross section additional output variables."""
+
+    ENCROACHMENT_STATION_LEFT = "Encroachment Station Left"
+    ENCROACHMENT_STATION_RIGHT = "Encroachment Station Right"
+    AREA_INEFFECTIVE_TOTAL = "Area including Ineffective Total"
+    VELOCITY_TOTAL = "Velocity Total"
+
+
+XS_STEADY_ADDITIONAL = [
+    XsSteadyAdditionalVar.ENCROACHMENT_STATION_LEFT,
+    XsSteadyAdditionalVar.ENCROACHMENT_STATION_RIGHT,
+    XsSteadyAdditionalVar.AREA_INEFFECTIVE_TOTAL,
+    XsSteadyAdditionalVar.VELOCITY_TOTAL,
+]
```

Comment:
* The `XS_STEADY_OUTPUT` and `XS_STEADY_ADDITIONAL` constants here are redundant since they both contain the full list of variables in each corresponding enum. The other lists of specific enum values above were created because there are places where only some of the enum values are relevant. You could either combine the enums into a single `XsSteadyOutputVar` enum and create an `XS_STEADY_OUTPUT_ADDITIONAL` list that contains only "additional" vars, or simply remove the `XS_STEADY_OUTPUT` and `XS_STEADY_ADDITIONAL` vars.